CHANGELOG.md

```diff
@@ -0,0 +1,178 @@
+
+[Full Changelog](https://github.com/arborx/arborx/compare/97bbec21cc92dd2b4bd3a68c52a230b4c3c4643c...v0.8-beta2)
+
+**New features:**
+
+- Provide access traits for predicates [\#117](https://github.com/arborx/ArborX/pull/117)
+- Add version\(\) and gitCommitHash\(\) [\#61](https://github.com/arborx/ArborX/pull/61)
+
+**Build changes:**
+
+- Replace TRIBITS build system with CMake [\#14](https://github.com/arborx/ArborX/pull/14)
+- Replace ArborX\_ENABLE with ARBORX\_ENABLE [\#99](https://github.com/arborx/ArborX/pull/99)
+- Provide an option to disable tests and examples [\#31](https://github.com/arborx/ArborX/pull/31)
+- Make MPI dependency optional [\#42](https://github.com/arborx/ArborX/pull/42)
+
+**Enhancements:**
+
+- Use MPI\_Comm\_dup to separate ArborX comm context from user's [\#135](https://github.com/arborx/ArborX/pull/135)
+- Add CMake option for enabling benchmarks [\#138](https://github.com/arborx/ArborX/pull/138)
+- Add intersects\(Point, Box\) [\#122](https://github.com/arborx/ArborX/pull/122)
+- Mark BVH::{size,empty,bounds} as noexcept [\#114](https://github.com/arborx/ArborX/pull/114)
+- Improve error messages in BVH constructor and BVH::query\(\) [\#113](https://github.com/arborx/ArborX/pull/113)
+- Make data private in Point [\#100](https://github.com/arborx/ArborX/pull/100)
+- Constexpr geometric primitives and algorithms [\#97](https://github.com/arborx/ArborX/pull/97)
+- Add CUDA+clang CI test [\#95](https://github.com/arborx/ArborX/pull/95)
+- Make sure the recorded git hash is recent [\#79](https://github.com/arborx/ArborX/pull/79)
+- Test installation of ArborX [\#48](https://github.com/arborx/ArborX/pull/48)
+- Relax CudaUVM requirement [\#24](https://github.com/arborx/ArborX/pull/24)
+- Find Boost in subdirectories that actually require it [\#22](https://github.com/arborx/ArborX/pull/22)
+
+**Fixed bugs:**
+
+- Fix parsing options in the ArborX benchmark [\#54](https://github.com/arborx/ArborX/pulls/54)
+- Fix compilation against Kokkos development branch [\#40](https://github.com/arborx/ArborX/pulls/40)
+- Optimize communication within the same rank [\#134](https://github.com/arborx/ArborX/pull/134)
+- Fix for distributed searches with large count of results per que… [\#129](https://github.com/arborx/ArborX/pull/129)
+- Fix Kokkos\_DEVICES spelling [\#35](https://github.com/arborx/ArborX/pull/35)
+- Add missing STATUS keyword in message [\#98](https://github.com/arborx/ArborX/pull/98)
+- Fix memory management in CmdLineArgs [\#66](https://github.com/arborx/ArborX/pull/66)
+- Fix memory violation when processing cmd line args in the driver [\#63](https://github.com/arborx/ArborX/pull/63)
+- Fix unused variable warnings [\#25](https://github.com/arborx/ArborX/pull/25)
+
+**Deprecations:**
+
+- Deprecate old aliases to generate predicates [\#119](https://github.com/arborx/ArborX/pull/119)
+- Disable deprecated code [\#40](https://github.com/arborx/ArborX/pull/40)
```

Comment:
```suggestion
```

Reply:
Why? I think deprecations should be in the changelog.

Reply:
Deprecation as compared to what.  Also the 2nd one is quit using deprecated Kokkos functionalities

Reply:
OK with removing the second one. First one is deprecation to the previous version.